test/e2e/integration/app-frontend/group.ts

```diff
@@ -193,8 +194,10 @@ describe('Group', () => {
   it('should support panel group adding item to referenced group', () => {
     init();
     cy.get(appFrontend.group.showGroupToContinue).find('input').check({ force: true });
-    cy.get(appFrontend.group.secondGroup_add).should('exist').and('be.visible').focus().click();
-    cy.get(appFrontend.group.secondGroup_add_to_reference_group).should('exist').and('be.visible').focus().click();
+    cy.get(appFrontend.group.secondGroup_add).should('exist');
+    cy.get(appFrontend.group.secondGroup_add).click();
```

Comment:
Fully agree with the cleanup, but we can go further. Cypress can't click something that doesn't exist, so we can omit the `should('exist')` line as well. It will try to find the element in order to click it before timing out if it doesn't exist.

Reply:
Thanks for feedback. I will definitely follow your suggestion and omit the should("exist"); ☺️